src/app/projects/[project_id]/page.tsx

```diff
@@ -163,6 +164,16 @@ export default async function Project({
 
   return (
     <Container maxWidth="lg" className="project-box">
+      <Button
+        startIcon={
+          <ArrowBackIosIcon className="project-box__back-button-icon" />
+        }
+        className="project-box__back-button"
+        href="./project"
+        variant="text"
+      >
+        Back to project
```

Comment:
```suggestion
        Back to All Projects
```

My suggestion is to just slightly improve on the grammar